noxfile.py

```diff
@@ -13,7 +13,7 @@
     "jinja2",
     "numpy==1.23.5",
     "typing_extensions",
-    "beartype",
+    "beartype!=0.16.0",
```

Comment:
```suggestion
    "beartype<0.16.0",
```
nit: or just smaller for now? Not sure whether they can fix the issue in the next version and if not, it will break again.

Reply:
I expect they will fix the issue in the next version because it is impacting many users. I will send a patch if that doesn't happen